cli/src/pcluster/api/controllers/cluster_instances_controller.py

```diff
@@ -28,10 +39,26 @@ def delete_cluster_instances(cluster_name, region=None, force=None):
 
     :rtype: None
     """
-    return None
+    cluster = Cluster(cluster_name)
+    try:
+        if not check_cluster_version(cluster):
+            raise BadRequestException(
+                f"cluster '{cluster_name}' belongs to an incompatible ParallelCluster major version."
+            )
+        if cluster.stack.scheduler == "awsbatch":
+            raise BadRequestException("the delete cluster instances operation does not support AWS Batch clusters.")
+    except StackNotFoundError:
+        if not force:
+            raise NotFoundException(
+                f"cluster '{cluster_name}' does not exist or belongs to an incompatible ParallelCluster major version. "
+                "To force the deletion of all compute nodes, please use the DeleteClusterInstances API "
```

Comment:
let's not mention DeleteClusterInstances API so that the error message is also applicable to the CLI. ...please use the force parameter

Reply:
Done